Symfony/CS/Tests/AbstractIntegrationTest.php

```diff
@@ -0,0 +1,260 @@
+    }
+
+    /**
+     * @param string $fileName
+     * @param string $config
+     *
+     * @return FixerInterface[]
+     */
+    protected function getFixersFromConfig($fileName, $config)
+    {
+        static $levelMap = array(
+            'none' => FixerInterface::NONE_LEVEL,
+            'psr1' => FixerInterface::PSR1_LEVEL,
+            'psr2' => FixerInterface::PSR2_LEVEL,
+            'symfony' => FixerInterface::SYMFONY_LEVEL,
+        );
+
+        $lines = explode("\n", $config);
+        if (count($lines) < 1) {
+            throw new InvalidArgumentException(sprintf('No configuration options found in "%s".', $fileName));
+        }
+
+        $config = array('level' => null, 'fixers' => array(), '--fixers' => array());
+
+        foreach ($lines as $line) {
+            $labelValuePair = explode('=', $line);
+            if (2 !== count($labelValuePair)) {
+                throw new InvalidArgumentException(sprintf('Invalid configuration line "%s" in "%s".', $line, $fileName));
+            }
+
+            $label = strtolower(trim($labelValuePair[0]));
+            $value = trim($labelValuePair[1]);
+
+            switch ($label) {
+                case 'level' : {
+                    if (!array_key_exists($value, $levelMap)) {
+                        throw new InvalidArgumentException(sprintf('Unknown level "%s" set in configuration in "%s", expected any of "%s".', $value, $fileName, implode(', ', array_keys($levelMap))));
+                    }
+
+                    if (null !== $config['level']) {
+                        throw new InvalidArgumentException(sprintf('Cannot use multiple levels in configuration in "%s".', $fileName));
+                    }
+
+                    $config['level'] = $value;
+                    break;
+                }
+                case 'fixers' : {
+                    $fixers = explode(',', $value);
+                    foreach ($fixers as $fixer) {
+                        $config['fixers'][] = strtolower(trim($fixer));
+                    }
+
+                    break;
+                }
+                case '--fixers' : {
+                    $fixers = explode(',', $value);
+                    foreach ($fixers as $fixer) {
+                        $config['--fixers'][] = strtolower(trim($fixer));
+                    }
+
+                    break;
+                }
+                default : {
+                    throw new InvalidArgumentException(sprintf('Unknown configuration item "%s" in "%s".', $label, $fileName));
+                }
+            }
+        }
+
+        if (null === $config['level']) {
+            throw new InvalidArgumentException(sprintf('Level not set in configuration "%s".', $fileName));
+        }
+
+        $fixer = new Fixer();
+        $fixer->registerBuiltInFixers();
+        $allFixers = $fixer->getFixers();
+
+        $fixers = array();
+        for ($i = count($allFixers) - 1; $i >= 0; --$i) {
+            /** @var FixerInterface $fixer */
```

Comment:
it is needed.
Fixer's method `getFixers` has no PHPDoc, not does the returned property;
```php
    protected $fixers = array();
```